tests/e2e/utils/common.go

```diff
@@ -368,12 +376,18 @@ func ExposeCloudService(name, serviceHandler string) error {
 
 //CreateServiceObject function to create a servcice object
 func CreateServiceObject(name string) *v1.Service {
+	var portInfo []v1.ServicePort
+	if !ProtocolQuic{
+		portInfo = []v1.ServicePort{{Protocol: "TCP", Port: 10000, TargetPort: intstr.FromInt(10000)}}
+	}else {
+		portInfo = []v1.ServicePort{{Protocol: "UDP", Port: 10001, TargetPort: intstr.FromInt(10001)}}
```

Comment:
devil number, it's sound not good

Reply:
@ugvddm , Thank you for reviewing the PR, can you please elobarate you comment, i could not understand your meaning of "devil number"...

Reply:
It's better to define a constant instance of using number in code

Reply:
> It's better to define a constant instance of using number in code

+1